draft-ietf-capport-architecture.xml

```diff
@@ -632,35 +632,45 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
                 to which the IP belongs.
           </t>
         </section>
-        <section anchor="context_free_url">
-          <name>Context-free URL</name>
+      </section>
+        <section anchor="context_free_uri">
+          <name>Context-free URI</name>
+          <t>
+            A Captive Portal API needs to present information to clients
+            that is unique to that client. To do this, some systems use
+            information from the context of a request, such as the source
+            address, to identify the UE.
+          </t>
           <t>
-	     The URLs provided in the API SHOULD contain all the information
-	     necessary to render the resources requested: the resources should
-	     not depend on ambient information, such as remote address on the
-	     connection. This is to ensure that the content served from these
-	     URLs is correct and meaningful to the User Equipment, even when
-	     accessed from a network other than the one that contains the
-	     captive portal.  One consequence of this is that URLs provided in
-	     the API are expected to be resolved using public global DNS (as
-	     defined in Section 2 of <xref target="RFC8499"/>).
+            Using information from context rather than information from the
+            URI allows the same URI to be used for different clients. However,
+            it also means that the resource is unable to provide relevant
+            information if the UE makes a request using a different network
+            path. This might happen when UE has multiple network interfaces.
+            It might also happen if the address of the API provided by DNS
+            depends on where the query originates (as in split DNS
+            <xref target="RFC8499"/>).
           </t>
           <t>
-	     Though a URL might still correctly resolve when the UE makes the
-	     request from a different network, it is possible that some
-	     functions could be limited to when the UE makes requests using the
-	     captive network. For example, payment options could be absent or a
-	     warning could be displayed to indicate the payment is not for the
-	     current connection.
+            Accessing the API MAY depend on contextual information. However,
+            the URIs provided in the API SHOULD be unique to the UE and not
+            dependent on contextual information to function correctly.
           </t>
           <t>
-	     URLs could include some means of identifying the User Equipment in
-	     the URLs.  However, including unauthenticated User Equipment
-	     identifiers in the URL may expose the service to spoofing or replay
-	     attacks.
+            Though a URI might still correctly resolve when the UE makes the
```

Comment:
These last two paragraphs are the same as before, just unindented, with URL replaced by URI (in anticipation of a merge conflict with https://github.com/capport-wg/architecture/pull/65).